scripts/initialize_dynamodb_oss.py

```diff
@@ -3,9 +3,14 @@
 
 from consoleme.config import config
 
-ddb = boto3.client(
-    "dynamodb", endpoint_url=config.get("dynamodb_server"), region_name="us-east-1"
-)
+if config.get("dynamodb_server"):
+	ddb = boto3.client(
+	    "dynamodb", endpoint_url=config.get("dynamodb_server"), region_name=config.region
+	)
+else:
+	ddb = boto3.client(
+        "dynamodb", region_name=config.region
+    )
```

Comment:
this shouldn't be necessary because config.get("dynamodb_server") will default to `None` if it is not set in the configuration. So essentially this conditional does the same thing as the code should do currently

Reply:
Oh, okay cool. I though we had to specifically omit that part in the code too.

Reply:
Oh, okay cool. I thought we had to specifically omit that part in the code too.